### ISSUE TITLE
GPU training not working with Deep learning AMI P3 instance

### ISSUE THREAD

**Issue by mohammedayub44** [tensorflow/nmt#397]
I'm using AWS Deep learning Ubuntu 15.0 P3 instance. For some reason my tensorflow_p36 environment is not able to detect the 4 GPU instances. 
I have ran the test gpu command and it return false.
`import tensorflow as tf`
`tf.test.is_gpu_available()` returns `False`

Below is the result of my `nvidia-smi` output:

![image](https://user-images.githubusercontent.com/7122670/46315187-bfe6a780-c59a-11e8-8763-cb7922f99d00.png)

I also tried the below command: 

`CUDA_VISIBLE_DEVICES=0,1,2,3 python -m nmt.nmt --src=en --tgt=es --vocab_prefix=/home/ubuntu/mukund_traildata/vocab --train_prefix=/home/ubuntu/mukund_traildata/new_train --dev_prefix=/home/ubuntu/mukund_traildata/new_dev --test_prefix=/home/ubuntu/mukund_traildata/new_testing --out_dir=/home/ubuntu/mukund_traildata/models/model3 --num_gpus=1 --num_train_steps=1200 --steps_per_stats=100 --num_layers=4 --num_units=128 --dropout=0.2 --metrics=bleu --log_device_placement=True`

There is no GPU action:

![image](https://user-images.githubusercontent.com/7122670/46315235-e99fce80-c59a-11e8-8d2a-c854c7dfab18.png)

Let me know if I'm missing any configurations. Appreciate any help.

Mohammed Ayub
